src/components/Countdown/Countdown.tsx

```diff
@@ -89,22 +91,24 @@ const Countdown = forwardRef<HTMLDivElement, CountdownProps>(
 
     useEffect(() => {
       const timer = setInterval(() => {
-        setCurrentDateTime(new Date());
+        setCurrentDateTime((currentTime) => {
+          return new Date(currentTime.getTime() + 1000);
```

Comment:
could we just assume that the consumer is changing the currentDateTimeProp themselves instead of managing internal state?

Reply:
think this is what actually moves the timer along, so probably don't want to reimplement in each consumer?